### ISSUE TITLE
Amplicon sequencing

### ISSUE THREAD

**Issue by HuGang-c** [grenaud/deML#22]
Hi, Thanks for soft, it looks like it can be used in amplicon sequencing, but I do not know how to make it right for the input files.

I have 12 different F primers, and another 8 different primers, I use the primer sequence to introduce barcodes for my sample, thus by combine the F and R primers, I can mix 96 samples one time.

Now I already get the paired sequencing data for 96 mixed samples, I made a index file like:

#Index1	Index2	Name
GAGTAGTA	TACTTATG	1A
GAGTAGTA	CGTTGCGA	1B
....                     ....                    1H
TTCGCTTG	TACTTATG	2A
TTCGCTTG	CGTTGCGA	2B
....                     ....                    2H
....                     ....                    ....
....                     ....                    12H

but deML quit with:
TACTTATG from A1 causes a conflict with A10 A11 A12 A2 A3 A4 A5 A6 A7 A8 A9

can you tell me what happen?

And I am also confused with the -if1 -if2 inputs, How should I prepare this two file in my condition?

**Comment by grenaud** [comment]
a collision is not the end of the world if the second index can differentiate it. 
I believe those are the index files for first index and second index, can you check the test data?

**Comment by HuGang-c** [comment]
Sorry I did not understand, maybe it is because I was not clearly asking

I have checked the test files, the problem is that I do not have -if1 -if2 inputs.

I know exactly my barcodes sequence, as I design it by myself

I am sequencing the PCR product which is amplified by my primers

Thus my barcode is in the sequence itself, and technically it will help me to know which sample they came from.

Can deML demultiplex my sequence in my condition?
